packages/grafana-data/src/types/data.ts

```diff
@@ -102,6 +103,8 @@ export interface Column {
   filterable?: boolean;
   unit?: string;
   custom?: Record<string, any>;
+  // To transfer data from dataFrame column
+  links?: DataLink[];
```

Comment:
this should no longer be needed

Reply:
as the table translation is no longer used for table in explore

Reply:
removed